### PR TITLE
docs: add docs for jcloud horizontal pod autoscale

### PR DESCRIPTION
**Pull request by winstonww**
**Goals:**
- Add docs for jcloud horizontal pod autoscale 

- [ ] check and update documentation. See [guide](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#-contributing-documentation) and ask the team.

### CODE REVIEW

File: docs/concepts/jcloud/configuration.md [hunk 1]
```diff
@@ -226,12 +226,13 @@ executors:
...
+| target           | 100         | int                                         | Target number the replicas try to maintain. If metric is `cpu`, this is the % CPU utilization (e.g. 60 means replicas will be scaled when pods on average reach 60% CPU utilization). If metric is `memory`, this is the target millibytes of memory (e.g. 200m means replicas will be scaled when each pod memory consumption exceeds `200m` on average). If metric is `rps`, this is the target number of total requests processed per second per replica. Lastly, if metric is `concurrency`, this is the target number of concurrent requests processed at any given time. |
```
Comment (tarrantro) [inline docs/concepts/jcloud/configuration.md:235]: we should elabroate the metrics target in below, not put too much information in the table.

**Review by tarrantro — DISMISSED** [review]
lgtm, please rebase the latest and merge

File: docs/concepts/jcloud/configuration.md [hunk 2]
```diff
@@ -226,15 +226,24 @@ executors:
...
+| metric           | concurrency | `concurrency` / `rps` / `cpu` / `memory`    | Metric for scaling                                                        |
+| scale_down_delay | 30s         | str, `0s` <= value <= `1h`                  | Time window which must pass at reduced concurrency before a scaling down  |
+| target           | 100         | int                                         | Target number the replicas try to maintain.                               |
+
+The unit of `target` depends of the metric specified. For `metric` equals:
+- `concurrency`: `target` refers to the number of concurrent requests processed at any given time.
+- `rps`: `target` refers to the number of requests processed per second per replica.
+- `cpu`: `target` refers to average % CPU utilization of each pod (e.g. `60` means replicas will be scaled up when pods on average reach 60% CPU utilization)
+- `memory`: `target` refers to average millibytes of memory used by each pod (e.g. `200` means replicas will be scaled up when the average pods' memory consumption exceeds 200m).
```
Comment (scott-martens) [inline docs/concepts/jcloud/configuration.md:241]:
```suggestion
The unit of `target` depends on the metric specified. Refer to the table below:
| Metric                | Target                                                                                            |
| --------------- | ------------------------------------------------------------ |
| `concurrency` | Number of concurrent requests processed at any given time. |
| `rps` | Number of requests processed per second per replica. |
| `cpu` | Average % CPU utilization of each pod<br>(e.g. `60` means replicas will be scaled up when pods reach an average 60% CPU utilization)
| `memory` | Average megabytes of memory used by each pod<br>(e.g. `200` means replicas will be scaled up when the average pod's memory consumption exceeds 200m).
```

File: docs/concepts/jcloud/configuration.md [hunk 3]
```diff
@@ -226,15 +226,24 @@ executors:
...
 
 After you make a JCloud deployment using the autoscaling configuration, the Flow serving part is just the same: the only difference you may notice is it takes a few extra seconds to handle the initial requests since it needs to scale the deployments behind the scenes. Let JCloud handle the scaling from now on, and you can deal with the code!
 
+Note, that if `metric` is `cpu` or `memory`, `min` will be reset to 1 if it is set to 0 by the user.
```
Comment (scott-martens) [inline docs/concepts/jcloud/configuration.md:245]:
```suggestion
Note, that if `metric` is `cpu` or `memory`, `min` will be reset to 1 if the user sets it to 0.
```

File: docs/concepts/jcloud/configuration.md [hunk 4]
```diff
@@ -226,15 +226,24 @@ executors:
...
+
 ### Pricing
 At present, pricing for autoscaled Executor/Gateway follows the same {ref}`JCloud pricing rules <jcloud-pricing>` for the most part.
```
Comment (scott-martens) [inline docs/concepts/jcloud/configuration.md:248]:
```suggestion
At present, pricing for autoscaled Executor/Gateway largely follows the same {ref}`JCloud pricing rules <jcloud-pricing>`.
```

**Pull request merged**